src/ui/public/timepicker/absolute_panel/kbn_timepicker_absolute_panel.html

```diff
@@ -12,7 +15,10 @@
             ng-click="setToNow({key:'from'})"
             kbn-accessible-click
           >
-            Set To Now
+            <FormattedMessage
```

Comment:
Actually `FormattedMessage` is not an Angular directive, that exists (only a React component). So putting FormattedMessage here works, but only because it's an unknown element to the browser, so it will treat it like a `span` by default for rendering. So we should replace that with either of two choices.

If it's the sole content of it's element (like in this case), just pull up the `i18n-xx` attributes to the encapsulating element, so in that case:

```html
<a
  class="kuiLocalDropdownHeader__action"
  ng-click="setToNow({ key: 'from' })"
  kbn-accessible-click
  i18n-id="common.ui.timepicker.setToNow"
  i18n-default-message="Set To Now"
/>
```

In the places were it's not the only element of it's child we should just replace the `FormattedMessage` by a `span` element.